draft-ietf-jsonpath-base.md

```diff
@@ -622,8 +625,8 @@ The syntax and semantics of each kind of selector are defined below.
 
 A name selector `'<name>'` selects at most one object member value.
 
-Applying the `name-selector` to an object value in its input nodelist,
-its string is required to match the corresponding member value.
+Applying the `name-selector` to an object node
+selects a member value whose name matches the provided name.
```

Comment:
Good.

Reply:
Adopted in 39a387e0583799b260f45184b3cd0e6ad01084a9 which also moves this text into the semantics section and tidies up that section.